### DISCUSSION TITLE
Skin editor in osu! lazer

### DISCUSSION THREAD

**Discussion by nexies** [ppy/osu#22011] [Ideas] [answered]
I've got an idea that implementing a sound editor in the skin editor might be very useful. The ability to change sounds of clicks, combo breaks, background music without going into directory and manually changing the files! 
What do you think about it?

**Comment by peppy** [comment, marked as answer]
Ctrl+Shift+S. More features will be coming online in the future. You can drag files into the screen for now to update them.
